.pipelines/3p-e2e.yml

```diff
@@ -46,7 +46,7 @@ resources:
         - repository: 1P
           type: git
           name: IDDP/msal-javascript-1p
-          ref: master
+          ref: rename_cert_path
```

Comment:
This ref and the rest of this file's changes will be reverted before merging into dev. This ref change is needed for pointing to the corresponding 1P change.